tree/ntuple/v7/doc/specifications.md

```diff
@@ -406,7 +410,7 @@ The flags field can have any of the following bits set:
 | 0x04     | Has ROOT type checksum as reported by TClass                               |
 
 If `flag==0x01` (_repetitive field_) is set, the field represents a fixed-size array.
-Another (sub) field with `Parent Field ID` equal to the ID of this field
+Usually, another (sub) field with `Parent Field ID` equal to the ID of this field
```

Comment:
Why only "usually"? If there is a specific exception, maybe better to spell it out?

Reply:
Will do., `std::bitset` is the exception.